specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/examples/ServerDevOpsAuditCreateMax.json

```diff
@@ -0,0 +1,37 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "devAuditTestRG",
+    "serverName": "devOpsAuditTestSvr",
+    "devOpsAuditPolicyName": "default",
+    "api-version": "2020-02-02-preview",
+    "parameters": {
+      "properties": {
+        "state": "Enabled",
+        "storageAccountAccessKey": "sdlfkjabc+sdlfkjsdlkfsjdfLDKFTERLKFDFKLjsdfksjdflsdkfD2342309432849328476458/3RSD==",
+        "storageEndpoint": "https://mystorage.blob.core.windows.net",
+        "storageAccountSubscriptionId": "00000000-1234-0000-5678-000000000000",
+        "isAzureMonitorTargetEnabled": true
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/devAuditTestRG/providers/Microsoft.Sql/servers/devOpsAuditTestSvr/devOpsAuditingSettings/default",
+        "name": "default",
+        "type": "Microsoft.Sql/servers/devOpsAuditingSettings",
+        "properties": {
+          "state": "Enabled",
+          "storageEndpoint": "https://mystorage.blob.core.windows.net",
+          "storageAccountSubscriptionId": "00000000-1234-0000-5678-000000000000",
+          "isAzureMonitorTargetEnabled": true
+        }
+      }
+    },
+    "202": {
+      "startTime": "2020-09-22T09:34:54.72Z",
+      "operation": "UpsertServerDevOpsAuditPolicy"
```

Comment:
Please follow the guideline here: https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#creatingupdating-using-put